packages/client/hmi-client/src/components/operator/tera-operator-menu.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+	<section ref="operatorMenu" :style="style">
```

Comment:
I would prefer an `nav` in that case.
```suggestion
	<nav ref="operatorMenu" :style="style">
```